organization/urls.py

```diff
@@ -25,6 +25,7 @@
 from django.conf.urls import include, url
 from django.conf.urls.i18n import i18n_patterns
 from django.contrib import admin
+import debug_toolbar
```

Comment:
the import should be conditional as the module is not loaded when `DEBUG=False`

see https://github.com/Ircam-Web/Mezzo/blob/master/app/urls.py#L157